internal/tutils/psql.go

```diff
@@ -0,0 +1,136 @@
+package tutils
+
+import (
+	"fmt"
+	"math/rand"
+	"os/exec"
+	"strings"
+	"time"
+
+	"github.com/google/uuid"
+
+	"github.com/osbuild/image-builder/internal/db"
+)
+
+type PSQLContainer struct {
+	name string
+	id   string
+	port int
+}
+
+const (
+	image string = "postgres:13"
+	user  string = "postgres"
+)
+
+func containerRuntime() (string, error) {
+	out, err := exec.Command("which", "podman").Output()
+	if err == nil {
+		return strings.TrimSpace(string(out)), nil
+	}
+	out, err = exec.Command("which", "docker").Output()
+	if err == nil {
+		return strings.TrimSpace(string(out)), nil
+	}
+	return "", err
```

Comment:
nitpick: I would add a message that no container runtime was found and that the user needs to have one.